index.html

```diff
@@ -6,8 +6,36 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="style.css">
     <title>Portfolio</title>
+    <link href="//netdna.bootstrapcdn.com/twitter-bootstrap/2.3.2/css/bootstrap-combined.no-icons.min.css" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" />
+
+
 </head>
-<body>
-    
+<body class="container">
+    <div class="toolbar">
+        <h2 class="logo">Wahid Shadab</h2>
+        <div class="menu">
+            <div class="menu_item"></div>
+            <div class="menu_item"></div>
+            <div class="menu_item"></div>
+        </div>
+    </div>
+    <section class="headline">
+        <div class="intro">
+            <h3 class="hey">Hey there <br> I'm Shadab</h3>
+            <p class="iam">I'm a Software Developer</p>
+            <p class="about">
+                I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hestiate to contact me.
+            </p>
+        </div>
+        <div class="social_media">
+            <div class="fa fa-linkedin fa-2x"></div>
+            <div class="fa fa-github fa-2x"></div>
+            <div class="fa fa-twitter fa-2x"></div>
+            <div class="fa fa-facebook fa-2x"></div>
+            <div class="fa fa-youtube fa-2x"></div>            
+        </div>
```

Comment:
- The list of icons uses a ul tag to group them, and li per icon; each icon has a link (a tag)